tensorflow/java/README.md

```diff
@@ -120,8 +122,8 @@ libraries will need to be built from source.
       //tensorflow/java:libtensorflow_jni
     ```
 
-The JAR (`libtensorflow.jar`) and native library (`libtensorflow_jni.so`) will 
-be in `bazel-bin/tensorflow/java`.
+The JAR (`libtensorflow.jar`) and native library (unix: `libtensorflow_jni.so` or mac os x:`libtensorflow_jni.dylib`) will 
```

Comment:
Let's use the terms and formatting used in the rest of the doc:

(`libtensorflow_jni.so` on Linux and `libtensorflow_jni.dylib` on OS X)